modules/space/translation/keplertranslation.cpp

```diff
@@ -304,6 +304,26 @@ glm::dvec3 KeplerTranslation::position(const UpdateData& data) const {
     return _orbitPlaneRotation * p;
 }
 
+glm::dvec3 KeplerTranslation::debrisPos(const double& time) const {
```

Comment:
Is there a reason to have a separate method for this, instead of the regular `position` method?

Reply:
Must have been an intention to do position differently but never happened. I changed this to just having one `position` method to be consistent with other Translations.

Reply:
Cool, thanks!